internet-search/mini_rag_agent_slides_web.ipynb

```
@@ -0,0 +1,506 @@
+    "        for r in ddgs.text(query, region='us-en', safesearch='moderate', max_results=k):\n",
+    "\n",
+    "            url = r.get('href') or r.get('url')\n",
+    "\n",
+    "            title = r.get('title')\n",
+    "\n",
+    "            if not url:\n",
+    "\n",
+    "                continue\n",
+    "\n",
+    "            try:\n",
+    "\n",
```

Ruff (0.14.7)

Do not catch blind exception: `Exception`

(BLE001)